lib/Launcher.js

```diff
@@ -122,7 +127,7 @@ class Launcher {
           process.kill(-chromeProcess.pid, 'SIGKILL');
         // Attempt to remove temporary profile directory to avoid littering.
         try {
-          removeSync(temporaryUserDataDir);
```

Comment:
This should stay sync: handlers for process `exit` are expected to complete synchronously.